incubator/fluentd/templates/deamonset.yaml

```diff
@@ -49,23 +65,29 @@ spec:
           - name: http-input
             containerPort: 9880
             protocol: TCP
-        livenessProbe:
```

Comment:
Because liveness probe required config to provide the endpoint for the check. 
Not all containers support such configuration.
Instead of that, I disabled `supervisor` mode, so pod will failed if something goes wrong.
https://github.com/kubernetes/charts/pull/5357/files#diff-6ec61ff69620bf74339bd25610e91021R20

Reply:
I don't see how that achieves the same ends.